src/components/ProjectPreview.tsx

```diff
@@ -24,7 +24,7 @@ const ProjectPreview = ({project, width=280, showDescription=false}: Props) => {
   return (
     <Box>
       <Box>
-        <Link href={`/project/${project.projectId}`} underline="hover">
+        <Link href={`/project/${project.id.split('-')[0]}/${project.projectId}`} underline="hover">
```

Comment:
I think it makes sense to add the `contract` field to the `GetProject` query in `useProject.tsx` in order to do `project.contract.address` instead of the splitting here

Reply:
addressed with [cc69df2](https://github.com/ArtBlocks/artblocks-engine-react/pull/10/commits/cc69df2d9aca3b7a9ce1e7f4dfd30fbc722eca3f)